models/home.ini

```diff
@@ -25,6 +25,16 @@ type = flow
 label = Body blocks
 type = flow
 
+[fields.gutter_top]
+label = Gutter top
+description = Markwdown
```

Comment:
I am hesitant of using rst on some places where we need to determine the level in an absolute way. On rst the level of a heading is relative to parsing the whole structure so:

```
Header level ?
--------------
```

On markdown on the other hand, its explicit. I will add a description to the model fields so we know what type to use.


```
## Header level 2
```